apps/src/applab/applab.js

```diff
@@ -1162,6 +1162,7 @@ Applab.execute = function () {
   var codeWhenRun;
   if (level.editCode) {
     codeWhenRun = studioApp.getCode();
+    Applab.currentExecutionLog = [];
```

Comment:
No, because you are allowed to Reset the program that you just ran and then press Finish (at which point the execution log should be examined). It needs to stay active as the log from the last time you ran.